### PR TITLE
test: add test to check if manyToOne relationship is added

### PR DESCRIPTION
**Pull request by craicoverflow**
I wanted to verify this so wrote a test which I think should pass but isn't. I don't have all the context so I could be wrong here. Take a look and see if this is expected or unexpected behaviour.

### CODE REVIEW

File: packages/datastore/cli/tests/createJsonSchema.test.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+"""
+@model
+@datasync
+"""
+type Note {
+  _id: GraphbackObjectID!
+  title: String!
+  description: String!
+  """
+  @oneToMany(field: 'noteComment', key: 'noteId')
```
Comment (wtrocki) [inline packages/datastore/cli/tests/createJsonSchema.test.ts:19]: I will also add one with `noteID` missing

**Comment by kingsleyzissou** [comment]
@craicoverflow Thanks for this. Let me fix the lint issues on my end first because I can't see the failing tests.

**Comment by kingsleyzissou** [comment]
@craicoverflow the issue is with the filter method I wrote. It was using the default Type name for filtering out the undesired properties. When you specified the fieldname here: `@oneToMany(field: 'noteComment', key: 'noteId')`, it broke things.

What I will do, is I will merge this into my branch and refactor, because this test _should_ pass.

**Review by kingsleyzissou — APPROVED** [review]

**Comment by craicoverflow** [comment]
Sure - request my review whenever you need me to take another look.

**Pull request merged**